packages/data/verify.js

```diff
@@ -0,0 +1,178 @@
+async function main() {
+  const query = `
+    query ($cursor: Cursor) {
+      following(request: {
+        address: "0x617Fd60a77297d3533D4abe9dC353626bdA5E171"
+        limit: 50
+        cursor: $cursor
+      }) {
+        items {
+          profile {
+            id
+            name
+            handle
+          }
+        }
+        pageInfo {
+          prev
+          next
+        }
+      }
+    }
+  `;
+
+  const url = 'https://api.lens.dev/';
+  const allResponses = [];
+
+  let cursor = null;
+  while (true) {
+    const fetch = await import('node-fetch');
+    const response = await fetch.default(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ query: query, variables: { cursor: cursor } })
+    });
+
+    if (!response.ok) {
+      console.log(`Error: ${response.status} - ${await response.text()}`);
+      break;
+    }
+
+    const responseData = await response.json();
+    const items = responseData.data?.following?.items || [];
+    if (!items.length) {
+      break;
+    }
+
+    allResponses.push(...items);
+    const pageInfo = responseData.data?.following?.pageInfo || {};
+    cursor = pageInfo.next;
+
+    if (!cursor) {
+      break;
+    }
+  }
+
+  if (allResponses.length) {
+    // Store all the response data in 'resp.json'
+    const fs = require('fs');
+    fs.writeFileSync('resp.json', JSON.stringify(allResponses, null, 2));
+
+    // Read the contents of the 'resp.json' file
+    fs.readFile('./resp.json', 'utf8', (err, data) => {
+      if (err) {
+        console.error('Error reading the resp.json file:', err);
+        return;
+      }
+
+      try {
+        const jsonData = JSON.parse(data);
+
+        // Extract the id and name properties from resp.json
+        const profiles = jsonData.map((entry) => {
+          const { id, name } = entry.profile;
+          return `'${id}', // ${name}`;
+        });
+
+        // Load the verified.ts file
+        fs.readFile('./verified.ts', 'utf8', (err, tsData) => {
+          if (err) {
+            console.error('Error reading the verified.ts file:', err);
+            return;
+          }
+
+          const regex =
+            /export const mainnetVerified = \[[\S\s]*?...mainnetLensTeamMembers,[\S\s]*?...mainnetStaffs\n];/;
```

Comment:
We don't wanna spread those two arrays, cuz by default we follow all people need to be verified

Reply:
I am doing this to append the new data into the verified array before getting the unique data.

either way I have to take it to consideration even if I want to do it with existing data that I get after parsing the response, it's just would have to do the same thing post I get the unique data.